server/schemas/schema.graphql

```diff
@@ -2,6 +2,7 @@ type Schema implements Node {
   id: ID!
   projectId: ID!
   fields: [SchemaField!]!
+  titleField: ID
```

Comment:
You can also add `isTitle: Boolean!` to SchemaField

Reply:
Done!

Reply:
Done!